### ISSUE TITLE
Tracking Mode - Possibly Misleading Documentation?

### ISSUE THREAD

**Issue by LukasFehring** [mlco2/codecarbon#484]
* CodeCarbon version: 2.3.2
* Python version: 3.8
* Operating System: Linux Ubuntu

### Description

Hey all,
I am currently working on integration of cpu-time and gpu-time into codecarbon, and I am wondering: Does the tracking mode effect anything except the RAM? As far as I can tell the only hardware file using the parameter is 
```python3
class RAM(BaseHardware):
```

### What I Did

Work through the implementation

**Comment by benoit-cty** [comment]
Yes, you're write. Tracking mode only work for RAM yet. The PR for CPU process mode exist but has not been merged.

Sorry for the delay of the answer.

**Comment by LukasFehring** [comment]
> Yes, you're write. Tracking mode only work for RAM yet. The PR for CPU process mode exist but has not been merged.
Thanks :)

> Sorry for the delay of the answer.
No worries - I hope you had a nice Christmas and had a good start of 2024
